xDrip/Scenes/Settings/AlertSettings/SettingsAlertRoot/SettingsAlertRootInteractor.swift

```diff
@@ -27,7 +27,30 @@ final class SettingsAlertRootInteractor: SettingsAlertRootBusinessLogic, Setting
     // MARK: Do something
     
     func doLoad(request: SettingsAlertRoot.Load.Request) {
-        let response = SettingsAlertRoot.Load.Response()
+        let response = createResponse()
         presenter?.presentLoad(response: response)
     }
+    
+    private func doUpdate() {
+        let response = createResponse()
+        presenter?.presentUpdate(response: response)
+    }
+    
+    private func createResponse() -> SettingsAlertRoot.Load.Response {
```

Comment:
You're using `Load` use case for update event, which is not correct. Leave `Load` empty (if nothing should be there) and create a separate `Update` use case with corresponding models. Call `presentUpdate` during load event. This would prevent code duplication.